bazelrc-preset.bzl

```diff
@@ -93,23 +94,29 @@ generate_preset = rule(
     },
 )
 
-def bazelrc_preset(name, out_file = None):
+def bazelrc_preset(name, out_file = None, **kwargs):
     """
     Creates a bazelrc preset file.
 
     Args:
         name: The name of the preset.
         out_file: The file to write the preset to. If not provided, the preset will be written to `{name}.bazelrc`.
+        **kwargs: Additional named arguments to pass to the `generate_preset` rule.
+            Common attributes (those in https://bazel.build/reference/be/common-definitions#common-attributes)
+            are propagated to the generated `write_source_file` rule as well.
     """
     if lt("6.0.0"):
         fail("bazelrc_preset requires Bazel 6 or later. You are running Bazel {}".format(version))
+
     generate_preset(
         name = name,
         out = "_{}.bazelrc".format(name),
-        strict = select({
+        # If strict was included in kwargs, use it. Otherwise, use the value from the flag.
+        strict = getattr(kwargs, "strict", select({
```

Comment:
If often end up doing `kwargs.pop("strict", default)` to avoid that this is also part of `**kwargs`. Is this solved by `getattr` over `kwargs.get`

Reply:
no you're right, it would get duplicated